src/SFA.DAS.Reservations.Api.AcceptanceTests/Steps/AccountLegalEntitiesSteps.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using SFA.DAS.Reservations.Api.Controllers;
+using SFA.DAS.Reservations.Data;
+using TechTalk.SpecFlow;
+
+namespace SFA.DAS.Reservations.Api.AcceptanceTests.Steps
+{
+    [Binding]
+    public class AccountLegalEntitiesSteps : StepsBase
+    {
+        private OkObjectResult _actualResult;
```

Comment:
stick this onto testdata or some other injected thing.

Reply:
good point! fixed! 👍